images/ubuntu-24.04.yaml

```diff
@@ -0,0 +1,15 @@
+name: "ubuntu:24.04"
```

Comment:
The name should be probably just implied by the yaml file name.

The name probably shouldn't contain `:`, as it can be easily confused with Docker Hub images